gk/main.c

```diff
@@ -1246,7 +1448,9 @@ gk_proc(void *arg)
 			rx_queue_back, tx_queue_front,
 			lcore, gk_conf);
 
-		process_cmds_from_mailbox(instance, gk_conf);
+		process_cmds_from_mailbox(lcore, instance, gk_conf);
+
+		rte_timer_manage();
```

Comment:
Won't this be called every time through this loop for each instance, in addition to being called in the LLS block? We were already looking to reduce the number of times it was called in the LLS block because it was expensive. I wonder if it's enough to have it just called in that block, and not at all here.

If not, we should at least reduce how often it is called in GK.

Reply:
Without `rte_timer_manage()` being called in the GK blocks, when is `gk_flow_tbl_scan()` called? Isn't it the case that when the LLS block calls `rte_timer_manage()`, `rte_timer_manage()` will only call timers associated to the lcore of the LLS block?

Once we make `gk_flow_tbl_scan()` incrementally scan the flow table, we may be better off using a solution similar to what we did for fragmented packets in the GT blocks, that is, we may not need timers at all.